keyboards/keychron/q2/ansi/info.json

```
@@ -1,23 +1,88 @@
+            {"matrix": [1, 9], "x": 140, "y": 15, "flags": 4},
+            {"matrix": [1, 10], "x": 154, "y": 15, "flags": 4},
+            {"matrix": [1, 11], "x": 169, "y": 15, "flags": 4},
+            {"matrix": [1, 12], "x": 184, "y": 15, "flags": 4},
+            {"matrix": [1, 13], "x": 202, "y": 15, "flags": 1},
+            {"matrix": [1, 14], "x": 224, "y": 15, "flags": 1},
+
+            {"matrix": [2, 0], "x": 6, "y": 30, "flags": 1},
+            {"matrix": [2, 1], "x": 26, "y": 30, "flags": 4},
+            {"matrix": [2, 2], "x": 40, "y": 30, "flags": 4},
+            {"matrix": [2, 3], "x": 55, "y": 30, "flags": 4},
+            {"matrix": [2, 4], "x": 70, "y": 30, "flags": 4},
+            {"matrix": [2, 5], "x": 84, "y": 30, "flags": 4},
+            {"matrix": [2, 6], "x": 99, "y": 30, "flags": 4},
+            {"matrix": [2, 7], "x": 114, "y": 30, "flags": 4},
+            {"matrix": [2, 8], "x": 129, "y": 30, "flags": 4},
+            {"matrix": [2, 9], "x": 143, "y": 30, "flags": 4},
+            {"matrix": [2, 10], "x": 158, "y": 30, "flags": 4},
+            {"matrix": [2, 11], "x": 173, "y": 30, "flags": 4},
+            {"matrix": [2, 13], "x": 196, "y": 30, "flags": 1},
+            {"matrix": [2, 14], "x": 224, "y": 30, "flags": 1},
+
+            {"matrix": [3, 0], "x": 9, "y": 45, "flags": 1},
+            {"matrix": [3, 2], "x": 33, "y": 45, "flags": 4},
+            {"matrix": [3, 3], "x": 48, "y": 45, "flags": 4},
+            {"matrix": [3, 4], "x": 62, "y": 45, "flags": 4},
+            {"matrix": [3, 5], "x": 77, "y": 45, "flags": 4},
+            {"matrix": [3, 6], "x": 92, "y": 45, "flags": 4},
+            {"matrix": [3, 7], "x": 106, "y": 45, "flags": 4},
+            {"matrix": [3, 8], "x": 121, "y": 45, "flags": 4},
+            {"matrix": [3, 9], "x": 136, "y": 45, "flags": 4},
+            {"matrix": [3, 10], "x": 151, "y": 45, "flags": 4},
+            {"matrix": [3, 11], "x": 165, "y": 45, "flags": 4},
+            {"matrix": [3, 13], "x": 185, "y": 45, "flags": 1},
+            {"matrix": [3, 14], "x": 209, "y": 49, "flags": 1},
+
+            {"matrix": [4, 0], "x": 2, "y": 60, "flags": 1},
+            {"matrix": [4, 1], "x": 20, "y": 60, "flags": 1},
+            {"matrix": [4, 2], "x": 39, "y": 60, "flags": 1},
+            {"matrix": [4, 6], "x": 94, "y": 60, "flags": 4},
+            {"matrix": [4, 10], "x": 147, "y": 60, "flags": 1},
+            {"matrix": [4, 11], "x": 162, "y": 60, "flags": 1},
+            {"matrix": [4, 12], "x": 176, "y": 60, "flags": 1},
+            {"matrix": [4, 13], "x": 195, "y": 64, "flags": 1},
+            {"matrix": [2, 12], "x": 209, "y": 64, "flags": 1},
+            {"matrix": [4, 14], "x": 224, "y": 64, "flags": 1}
+
+        ]
     },
     "matrix_pins": {
         "cols": ["C14", "C15", "A0", "A1", "A2", "A3", "A4", "A5", "A6", "A7", "B0", "B1", "A8", "A9", "H3"],
         "rows": ["B4", "B3", "A15", "A14", "A13"]
     },
```

```suggestion
    "matrix_pins": {
        "cols": ["C14", "C15", "A0", "A1", "A2", "A3", "A4", "A5", "A6", "A7", "B0", "B1", "A8", "A9", "H3"],
        "rows": ["B4", "B3", "A15", "A14", "A13"]
    },
```
The mattix_pins definiton needs be deleted.

---

Is there a reason this should be done, even though this PR is move almost everything to `info.json`?

---

Hi, because it has the same definition as the file from the previous level, and other keyboards don't have it, it looks inconsistent. However, it's not necessarily something that needs to be deleted. Thanks~

---

Aaah. I see. I mistaken that change to be for `keychron/q2/info.json` instead of `keychron/q2/ansi/info.json`.